releasenotes/notes/ci_visibility-feat-intelligent_test_runner_ga-29333edb3933a74b.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    CI Visibility: Intelligent Test Runner is now GA, and can be disabled via DD_CIVISIBILITY_ITR_ENABLED. The Datadog
```

Comment:
do we have a link that we could include to docs or provide more info around this feature?

Reply:
We've actually decided not to announce the GA at this point, so I'll update the release note (we're still making the change to be `true` by default, though).